lib/breakers/statsd_plugin.rb

```diff
@@ -45,22 +45,19 @@ def filtered_endpoint_tag(path)
       uuids = /[a-fA-F0-9]{8}(\-?[a-fA-F0-9]{4}){3}\-?[a-fA-F0-9]{12}/
       institution_ids = /[\dA-Z]{8}/
       provider_ids = /Providers\(\d{10}\)/
-      v1_user_ids_1 = /[0-9a-f]{22}$/ # e.g. /api/v1/users/bb9c0f499977be68611151
-      v1_user_ids_2 = /00u[0-9a-zA-Z]{17}/ # e.g. /api/v1/users/00u2gskfa6kXUvU0N792
-
+      okta_users = %r{(api\/v1\/users\/)\w*}
       r = %r{
         (\/)
-        (#{digit}
+        (#{okta_users}
+        |#{digit}
         |#{contact_id}
         |#{uuids}
         |#{institution_ids}
-        |#{provider_ids}
-        |#{v1_user_ids_1}
-        |#{v1_user_ids_2})
+        |#{provider_ids})
         (\/|$)
       }x
 
-      path.gsub(r, '\1xxx\5')
+      path.gsub(r, '\1\3xxx\6')
```

Comment:
Can you explain the `\1\3xxx\6` part to me? I never quite understood this part in the code.

Reply:
yeah, it's in need of a serious comment/refactor

\1 refers to the first capture group and \6 refers to the 6th capture group.
a capture group is just the stuff matching the regex in parentheses.  So, it's just a complicated search and replace.

falling back to trusty https://rubular.com/
<img width="732" alt="Screen Shot 2020-02-24 at 5 18 51 PM" src="https://user-images.githubusercontent.com/19188/75196477-10c7e480-572a-11ea-9a91-17808bc0a397.png">